rllib/agents/impala/impala.py

```diff
@@ -587,15 +581,24 @@ def training_iteration(self) -> ResultDict:
 
         self.concatenate_batches_and_pre_queue(batch)
         self.place_processed_samples_on_learner_queue()
-        learner_results = self.process_trained_results()
+        train_results = self.process_trained_results()
 
         self.update_workers_if_necessary()
 
         # Callback for APPO to use to update KL, target network periodically.
         # The input to the callback is the learner fetches dict.
-        self._after_train_step(learner_results)
+        self.after_train_step(train_results)
```

Comment:
Since we have the training iteration fn now, we can also overwrite on APPO:

```
def training_teration(self):
   train_results = super().training_teration()
   self.after_train_step(train_results)
   return train_results
```
  
  Seems like the cleaner solution to me!

Reply:
I think this is done. APPO shares the same `training_iteration` fn with IMPALA and only overrides `self.after_train_step()`